### DISCUSSION TITLE
SO 2.3.21 install freezes at post installation scripts

### DISCUSSION THREAD

**Discussion by wtl06639** [Security-Onion-Solutions/securityonion#2732] [Unsupported Versions] [answered]
I am trying to install the Security Onion 2.3.21 version but it seems to get as far as the Running Post-installation scripts and then stops. If i reboot it gets back to a localhost login page but he username and password I created do not work so i am stuck. Anyone else running into this issue?

**Comment by weslambert** [comment]
You may want to try verifying the ISO, and what type of hardware/boot mode are you using?

**Reply by wtl06639** [reply]
I verified the iso. Looks correct. Im doing a onetime boot. The iso is connected via KVM. I let it sit at Running post-installtion scripts for over an hour but not its stuck at Pane is Dead so will no to go manually restart the server. This is on a Dell R840 server.  When I tried using the iso on USB it was saying i needed to enable UEFI mode, but i did that it would just bring me to a grub prompt. I can try and do the usb using the balena etcher tomorrow when i try the USB again.

**Comment by TOoSmOotH** [comment]
The post installation is copying a large tarball to the system. How long did you wait? You can alt+f2 to go into the /sysimage/nsm/docker/ and watch the filesize grow.

**Reply by wtl06639** [reply]
I waited over an hour. I can cehck the size doing alt+f2 tomorrow when i restart the server and try again. Currently sitting at a Pane is dead message and restart.

**Comment by SeanSpec** [comment, marked as answer]
I had this exact problem. The issue is that the virtual media manager for the iKVM only support a 7.9GB ISO and the SO image is bigger than that. I had to go on-site and use a USB stick.

**Reply by vdr0011** [reply]
the iso image is only 7.46GB

**Comment by gangalal** [comment]
Any solution for this?

**Reply by CyberMayler** [reply]
I was experiencing the same problem when installing directly through VMWARE and in addition to the management interface having internet access to download all packages, I had to change the boot type to UEFI in Settings -> Options -> Advanced - Firmware Type -> UEFI.
It was only after that that I no longer had problems with the post installation stuck using VMWARE. I did the same test on my proxmox and the result was the same for version 2.4 of Security Onion.

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/54895599/438090cb-f52d-4727-a2bf-1d770f9e4415)

**Comment by zura370** [comment]
> I am trying to install the Security Onion 2.3.21 version but it seems to get as far as the Running Post-installation scripts and then stops. If i reboot it gets back to a localhost login page but he username and password I created do not work so i am stuck. Anyone else running into this issue?



File name: 2.4.60-20240320 ISO 12gb file. I had the same problem with the latest one and tried all possibilities, but I didn't try the USB one. Help, I am struck!

![image](https://github.com/Security-Onion-Solutions/securityonion/assets/158118815/9a4d4303-1e15-4f99-8409-e91485448de4)

**Comment by oklier** [comment]
Hello,
I have the same issue while trying the setup/install a production node via an iLO card. Any updates to this? Seems like a common issue? I would hate to go to the data center with a USB if it can be corrected remotely. Also would hate to install oracle Linux and then the Onion. I need to be able to install remotely since the next node is over 1K miles away.
Thoughts?

**Reply by TotieBash** [reply]
Blu-ray is another option if you really don't want usb drive.
PXE boot is another option as well.
I also have mixed results using ILO or IDRAC.  My SO is a vm on ESXi so I mount the large iso on the vm level so I don't see this issue a lot.  If it was me and I am restricted from using usb thumb drive then I would use bluray disc.

**Reply by oklier** [reply]
Thanks for the reply. I finally got it installed but rebooting the system after about an hour and it booted just fine.

**Reply by osmanh21** [reply]
Hi I have this same issue right now. I am using Centos 7 64 bit. can you help me?

**Comment by oklier** [comment]
Hello,
For me it was just wait like an hour.

On Tue, May 21, 2024 at 12:33 PM osmanh21 ***@***.***> wrote:

> Hi I have this same issue right now. I am using Centos 7 64 bit. can you
> help me?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Security-Onion-Solutions/securityonion/discussions/2732#discussioncomment-9512177>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADAIFVFFOVYX5II4433WUP3ZDOHQ3AVCNFSM4WST36ZKU5DIOJSWCZC7NNSXTOKENFZWG5LTONUW63SDN5WW2ZLOOQ5TSNJRGIYTONY>
> .
> You are receiving this because you commented.Message ID:
> <Security-Onion-Solutions/securityonion/repo-discussions/2732/comments/9512177
> @github.com>
>


-- 
Otakar Klier
408.464.6109

**Comment by reyesj2** [comment]
This discussion is for 2.3.x which is EOL. If you're having trouble with Security Onion 2.4.x please create a new discussion with information about your deployment and any troubleshooting you've done. https://docs.securityonion.net/en/2.4/trouble-booting.html#booting-issues
